plugins/topology/src/components/Topology/utils/resource-utils.ts

```diff
@@ -0,0 +1,104 @@
+import * as _ from 'lodash';
+import { DeploymentModel, PodModel } from '../models';
+import { Alert, Alerts } from '../types/monitoring-types';
+import { OverviewItem } from '../types/resource';
+import { K8sModel, K8sResourceKind } from '../types/types';
+import { WORKLOAD_TYPES } from './topology-utils';
+
+export const MONITORABLE_KINDS = [
+  'Deployment',
+  'DeploymentConfig',
```

Comment:
What happens if the plugin is unable to locate resource type on the cluster? Does it raise errors/exceptions? I haven't run the plugin against vanilla Kubernetes myself, I'm just curious. I'm asking on the basis that `DeploymentConfig`s are not Kubernetes resources.

Reply:
yeah, have removed DeploymentConfig and this is the initial PR here we are looking for `'deployments', 'pods', 'services', 'ingresses', 'replicasets'` and node workload is formed for Deployment, stadalone Pod others resources are needed to form relationship i.e pods and replicasets to show deployment. There would be follow-up PRs to add a decorator for ingress on workload and sidebar which will open on click to show other associated resources, labels, annotations, etc